test/clangd_tests.vim

```diff
@@ -113,8 +113,8 @@ def g:Test_LspFormat()
 
   # file without an LSP server
   edit a.raku
-  assert_equal(['Error: Language server for "raku" file type is not found'],
```

Comment:
I would like to merge this PR.  Can you revert the changes to the clangd_tests.vim file that changes assert_equal() to assert_match()?

Reply:
is impossible to be `equal`? it had prefix and type now.

Reply:
it had uniformed the prefix/type. that's why i made this change. it is impossible to be `equal`.
// wish you're not to against my every PR. -_-#